### PR TITLE
feat(template): new comments with form - INNO-896

### PR DESCRIPTION
**Pull request by raduchiriac**
# PR description

Creates a new template that lists a few comments with a form to sumbit a new comment

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

* [ ] `package.json` is up-to-date and `@ec-europa/ecl-base` is part of the dependencies
* [ ] I have checked the dependencies
* [ ] I have given the fractal status “ready” to my component
* [ ] I have declared `@define mycomponent` in the SCSS file
* [ ] I have specified `margin: 0;` on the CSS component
* [ ] I have provided tests
* [ ] I follow the naming guidelines
* [ ] the component supports composition
* [ ] there are no hardcoded strings (all content come from the context)
* [ ] I have filled the README.md file (at least a few lines)
* [ ] My component is listed in the root README
* [ ] My PR has the right label(s)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 10c5f2e40020795d15818e85e9e279edae69ba9b

https://deploy-preview-548--europa-component-library.netlify.com

File: framework/templates/ecl-comments-with-form/ecl-comments-with-form.twig [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    - "operations" (object): object of operations (reply, edit, delete)
+      - "delete|edit|reply" (object):
+        - href (string)
+        - title (string)
+        - visible (boolean)
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-row ecl-comments-with-form' %}
```
Comment (yhuard) [inline framework/templates/ecl-comments-with-form/ecl-comments-with-form.twig]: - `ecl-comments-with-form` doesn't exist
- `ecl-row` should not be used here (no cols)

File: framework/templates/ecl-comments-with-form/ecl-comments-with-form.twig [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  {% set _css_class = _css_class ~ ' ' ~ extra_class %}
+{% endif %}
+
+{% if extra_attributes is defined %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+<div class="{{ _css_class }}" {{ _extra_attributes }}>
```
Comment (yhuard) [inline framework/templates/ecl-comments-with-form/ecl-comments-with-form.twig]: We don't need to encapsulate the code in a `div` (no class, no extra_attributes)

File: framework/templates/ecl-comments-with-form/ecl-comments-with-form.twig (hunk 1 above)
Comment (yhuard) [reply framework/templates/ecl-comments-with-form/ecl-comments-with-form.twig]: Please remove `_css_class` and `_extra_attributes` altogether. This is an example template, it doesn't need to be extensible ;)

File: framework/templates/ecl-comments-with-form/ecl-comments-with-form.twig [hunk 3]
```diff
@@ -0,0 +1,90 @@
+{#
```
Comment (yhuard) [inline framework/templates/ecl-comments-with-form/ecl-comments-with-form.twig]: Missing disclaimer:

```
{#
  This is an example and should not be used as it is.
#}
```

**Comment by drupol** [comment]
\o/

**Pull request merged**